schema/deploy/shipping/views.sql

```diff
@@ -2328,6 +2328,196 @@ grant select
   to "incidence-modeler";
 
 
+create or replace view shipping.scan_encounters_with_best_available_vaccination_data_v1 as
+
+    with vac_1_most_frequent_valid_date as (
+      select distinct on (individual)
+      individual,
+      vac_date,
+      vac_name_1
+      from shipping.scan_encounters_v1
+      where vac_date >= '2020-12-01'
+        and vac_date::date <= encountered::date
+      group by individual, vac_date, vac_name_1
+      having count(*) > 1
+      order by individual, count(*) desc
```

Comment:
Would adding another order by encountered make the ordering more deterministic? I can imagine someone entering, say, two different valid date twice.